docs/admin/configure.md

```diff
@@ -14,6 +14,25 @@ should not be localhost.
 If an access URL is not specified, Coder will create
 a publicly accessible URL to reverse proxy your deployment for simple setup.
 
+## Address
+
+You can change which port(s) Coder listens on. 
+
+```sh
+# Listen on port 80
+export CODER_HTTP_ADDRESS=0.0.0.0:80
```

Comment:
Here you are using `0.0.0.0:80` but in L77 you are using `127.0.0.1:3000`

This is a confusion.